### PR TITLE
adding support for power on travis builds + updating "dumb-init" version

### PR DESCRIPTION
**Pull request by ghatwala**
Here am trying to add support for power ("ppc64le") arch to travis-ci builds , Also i have updated the "dumb-init" version to latest 1.2.1 for "debian/debian-onbuild" dockerfiles.

Also travis-job created using this change - https://travis-ci.org/ghatwala/fluentd-docker-image/builds/389559309

Relevant git issue - #120

@repeatedly - FYI , please review/comment.

Signed-off-by: ghatwala <ghatwala@us.ibm.com>

### CODE REVIEW

**Comment by ghatwala** [comment]
@repeatedly  - could you please review ?

**Comment by ghatwala** [comment]
@repeatedly  - could you please review ?

**Comment by repeatedly** [comment]
Please edit template file, not generated file.

https://github.com/fluent/fluentd-docker-image/blob/master/Dockerfile.template.erb#L15

**Comment by ghatwala** [comment]
@repeatedly - thanks for the review comments, new commit (https://github.com/fluent/fluentd-docker-image/pull/123/commits/56dd77b48fbefaad974e69b3869546307d6406eb) -  added to this PR.

Can you please check now if everything seems fine ?

**Comment by ghatwala** [comment]
@repeatedly  - any review comments ?

**Comment by repeatedly** [comment]
Code looks good and need `make src-all` again

**Comment by ghatwala** [comment]
thanks @repeatedly  , lemme know if anything more is needed from my side before we can merge this commit ?

**Comment by repeatedly** [comment]
No need more work. Apply template changes by `make src-all` is all.

**Comment by ghatwala** [comment]
@repeatedly - done , could you please have a look 58e1c0365f7e7b708f5b838bcaf33c987b7fe920

File: Dockerfile.template.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@ FROM debian:stretch-slim
 LABEL maintainer "TAGOMORI Satoshi <tagomoris@gmail.com>"
 LABEL Description="Fluentd docker image" Vendor="Fluent Organization" Version="1.1"
 
-ENV DUMB_INIT_VERSION=1.2.0
+ENV DUMB_INIT_VERSION=1.2.1
```
Comment (repeatedly) [inline Dockerfile.template.erb]: From the test, alpine linux 3.7's dumb-init package is 1.2.0.
Need `if` condition like below:

```
<% if is_alpine %>
ENV DUMB_INIT_VERSION=1.2.0
<% else %>
ENV DUMB_INIT_VERSION=1.2.1
<% end %>
```

Comment (ghatwala) [reply Dockerfile.template.erb]: @repeatedly - done , add these changes ,thanks for the review

**Comment by repeatedly** [comment]
Merged. Thanks!

**Pull request merged**